hashring/hashring.go

```diff
@@ -64,7 +64,7 @@ func (r *HashRing) emit(event interface{}) {
 	}
 }
 
-// RegisterListener adds a listener that will be sent swim events.
+// RegisterListener adds a listener that will listens for HashRing events.
```

Comment:
Actually, scratch that. I just looked more closely at the events and you're right. A listener does receive all event types. My bad. Just the typo then: `s/listens/listen`! :smile: